ubclaunchpad/ubc-degree-navigator
# Add Docker file

## Describe your changes
> Add a short description of the changes. These will be the things to check for when the reviewers review your PR.
Added dockerfiles to facilitate deployment
#### Screenshot (if applicable)
> This can be FE screenshot, DB screenshot, API call from network tab, etc.

## How do I test
> Provide breif steps on how to test these changes. 
1.  Build the docker image using the command: docker build -t [name-of-image] .
2. Run the docker container using the command: docker run -p 8080:8080 [name-of-image]
The backend should then run as expected.

## Checklist before requesting a review
